src/script/components/Conversation/ConversationCells/CellsPagination/CellsPagination.tsx

```diff
@@ -0,0 +1,175 @@
+
+  const onSizeChange = useCallback(
+    (value: {value: string; label: string}) => {
+      setPageSize(parseInt(value.value));
+    },
+    [setPageSize],
+  );
+
+  const renderPageList = () => {
+    const lastPageIndex = numberOfPages - 1;
+    const spanLength = 1;
+    const endLength = 1;
+    const skipLength = 1;
+    const normalizeCount = endLength + skipLength + spanLength;
+
+    const dots = (key: string) => (
+      <Bold css={numberStyles} key={key}>
+        …
+      </Bold>
+    );
+
+    const renderPageNumber = (pageIndex: number) =>
+      currentPage === pageIndex ? (
+        <Bold key={pageIndex} css={{...numberStyles, ...numberActiveStyles}} data-uie-name="status-active-page">
+          {pageIndex + 1}
+        </Bold>
+      ) : (
+        <Link css={numberStyles} key={pageIndex} onClick={() => goPage(pageIndex)} data-uie-name="go-page">
+          {pageIndex + 1}
+        </Link>
+      );
+
+    const normalizedCurrent = Math.min(Math.max(currentPage, normalizeCount), lastPageIndex - normalizeCount);
+    const beforeCount = normalizedCurrent - spanLength - endLength;
+    const afterCount = lastPageIndex - endLength - normalizedCurrent - spanLength;
+
+    const pages = Array.from({length: numberOfPages}, (_, index) => renderPageNumber(index));
+
+    if (afterCount > skipLength) {
+      pages.splice(normalizedCurrent + spanLength + 1, afterCount, dots('dots-end'));
+    }
+    if (beforeCount > skipLength) {
+      pages.splice(endLength, beforeCount, dots('dots-start'));
+    }
+
+    return pages;
+  };
+
+  const options = [
+    {value: '10', label: '10'},
+    {value: '20', label: '20'},
+    {value: '50', label: '50'},
+    {value: '100', label: '100'},
+  ];
+  const crtValue = {value: `${pageSize}`, label: `${pageSize}`};
+
+  // @ts-ignore
+  return (
+    <FlexBox css={containerStyles}>
+      <div style={{flex: 1}}>
+        {firstRow}-{lastRow} out of {totalRows}
+      </div>
+      {numberOfPages > 1 && (
+        <FlexBox css={pagesContainerStyles} align="flex-end" data-uie-name="element-pagination" {...props}>
+          <div className={'previous-page'}>
+            {!isFirstPage && (
+              <IconButton
+                onClick={() => goPage(currentPage - 1)}
+                data-uie-name="go-previous-page"
+                css={arrowButtonStyles}
+                title={'Previous page - TODO TRANSLATE ME'}
+              >
+                <Icon.ArrowNextIcon css={arrowPreviousIconStyles} />
+              </IconButton>
+            )}
+          </div>
+
+          <div className={'list-pages'} data-uie-name="list-pages">
+            {renderPageList()}
+          </div>
+
+          <div className={'next-page'}>
+            {!isLastPage && (
+              <IconButton
+                onClick={() => goPage(currentPage + 1)}
+                data-uie-name="go-next-page"
+                css={arrowButtonStyles}
+                title={'Next page - TODO TRANSLATE ME'}
```

Comment:
To add translations:

1. Open `en-US.json`
2. Search for `cellGlobalView`, and add your strings there
3. Make sure your server is running
4. Import the translation function `import {t} from 'Util/LocalizerUtil'`
5. Use it like that - `t('yournamehere')`

Reply:
added strings to i18n